ts/features/euCovidCert/screens/valid/EuCovidCertValidScreen.tsx

```diff
@@ -146,12 +146,8 @@ const addBottomSheetItem = (config: {
     <View style={styles.flexColumn}>
       <View style={styles.row}>
         <View style={IOStyles.flex}>
-          <H3 color={"bluegreyDark"} weight={"Semibold"}>
-            {config.title}
-          </H3>
-          <H5 color={"bluegrey"} weight={"Regular"}>
-            {config.subTitle}
-          </H5>
+          <H4 color={"bluegreyDark"}>{config.title}</H4>
```

Comment:
Why in that case are we using a different mapping logic (`H3` → `H4` and `H5` → `H6`)?

Reply:
Make sense to me